wallaby.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (wallaby) {
```

Comment:
Wallaby was used while development on this feature as an experiment. Now it's expired, and I don't know if it's worth its price.